packaging_automation/update_docker.py

```diff
@@ -103,48 +105,29 @@ def update_changelog(project_version: str, exec_path: str, postgres_version: str
             raise ValueError(f"Already using version {project_version} in the changelog")
 
 
-def update_all_docker_files(project_version: str, exec_path: str, postgres_version: str):
+def update_all_docker_files(project_version: str, exec_path: str):
     template_path = f"{BASE_PATH}/templates/docker"
     pkgvars_file = f"{exec_path}/pkgvars"
 
-    if postgres_version:
-        update_pkgvars(project_version, template_path, pkgvars_file, postgres_version)
+    postgres_14_version, postgres_13_version, postgres_12_version = read_postgres_version(pkgvars_file)
 
-    persisted_postgres_version = read_postgres_version(pkgvars_file)
+    latest_postgres_version = postgres_14_version
 
-    update_docker_file_for_latest_postgres(project_version, template_path, exec_path, persisted_postgres_version)
+    update_docker_file_for_latest_postgres(project_version, template_path, exec_path, latest_postgres_version)
     update_regular_docker_compose_file(project_version, template_path, exec_path)
-    update_docker_file_alpine(project_version, template_path, exec_path, persisted_postgres_version)
-    update_docker_file_for_postgres12(project_version, template_path, exec_path)
-    update_docker_file_for_postgres14(project_version, template_path, exec_path)
-    update_changelog(project_version, exec_path, postgres_version)
+    update_docker_file_alpine(project_version, template_path, exec_path, latest_postgres_version)
+    update_docker_file_for_postgres12(project_version, template_path, exec_path, postgres_12_version)
+    update_docker_file_for_postgres13(project_version, template_path, exec_path, postgres_13_version)
+    update_changelog(project_version, exec_path, latest_postgres_version)
 
 
-def read_postgres_version(pkgvars_file: str) -> str:
+def read_postgres_version(pkgvars_file: str) -> Tuple[str, str, str]:
```

Comment:
I think you should change the name of this function now. It returns three versions, not a single one.

Reply:
Fixed as you suggested